tests/integration/testlib/statelib.py

```diff
@@ -227,6 +227,8 @@ def _convert_lag_numeric_value_options_to_integer(self):
                     for option_name, option_value in options.items():
                         with contextlib.suppress(ValueError):
                             option_value = int(option_value)
+                        if option_name == "arp_ip_target":
+                            option_value = option_value.replace(",", " ")
```

Comment:
Why does this need to be normalized in the test specific code? Shouldn't the Nmstate code properly handle the format differences?

Reply:
As is a difference in the verification phase, we need to also normalize for verification when using assert_state_match(). If not, the desired state won't match against the current state.

Reply:
Not sure I follow. Do you want to make Nmstate accept both `"10.1.1.2,10.1.1.3"` and `"10.1.1.2 10.1.1.3"` in the desired state? IMHO it should accept only one of those formats and also show only one of those formats. Then not adjustments for verification should be required.

Reply:
@EdDev @cathay4t what do you think?

Reply:
I agree with you. It does not make sense to accept both formats or accept one and then shows another.